vignettes/bcEpiRate.Rmd

```diff
@@ -384,17 +384,27 @@ popn_06 <- population %>%
 std_rt_06 <- death_rates %>%
   filter(disease == "Disease D") %>%
   pull(std_rt)
-  
-# calculate SMR
-smr <- get_smr(count = count_06, popn = popn_06, std_rt = std_rt_06)
-smr
 ```
 
-The number of deaths in the study is `r round(smr * 100, 1)`% of the
-number of expected deaths.
+Let's construct a 95% confidence interval around the estimate:
+
+```{r example-06-get-smr}
+df_06 <- get_smr(
+  count = count_06, popn = popn_06, std_r = std_rt_06,
+  std_measure = "rate", dist = "normal", interval = 0.95
+)
+df_06
+```
+
+```{r example-06-text, include=FALSE}
+# store all values for summary
+df_06 <- df_06 %>%
+  mutate(across(.cols = c(smr, lower, upper), ~ round(.x, 2)))
+```
 
-The ability to construct confidence intervals will be available in the
-future.
+The SMR for Disease D in 2016 was `r df_06$smr`; 95% CI
+[`r df_06$lower`, `r df_06$upper`]. The number of deaths in the study
+was approximately `r df_06$smr * 100`% of the expected number of deaths.
 
```

Comment:
I'm leaving a comment here but it applies to different places of this vignette, and am not sure if we want to address it or not (as some R users may be more used to this than I am :) ). 

I noticed that across different examples, where we are showing a glimpse into various data frames (tibbles), and the column types are double, using `head`, it seems to display the floor value of numbers, followed by a `.` to denote that they're decimals (double). I think I prefer the way in which `head` displays data frames (though it doesn't show variable types), but at least the numbers seem more aligned and consistent. Not super important in the grand scheme of things, but was wondering of the motivation for using tibble to store the data frames in these examples.

Reply:
I think choice of `tibble` is mostly to use `tidyverse` objects and the (subjective) tidyverse opinion of this being a better way to glimpse data. Personally, I do prefer it because these glimpses are meant to show the data structure and example values, not exact values for further calculation. So truncating the decimals is fine by me to improve readability. It's much easier, to me, to display "2." and "200." so that it's easier for me to see that one value is 100x larger than the other at a glance instead of having to parse the difference between something like "2.013" and "201.3". But am I misunderstanding?

Reply:
This is, to a degree, a subjective issue (that is, in the other format, I get more value from the fact that the numbers are right aligned, and from the way in which they're represented, whereas other users may prefer something else), so things can stay as they are. I appreciate the feedback.

Reply:
Changes not required. Resolving conversation.